packages/flutter_test/lib/src/matchers.dart

```diff
@@ -466,10 +466,10 @@ Matcher coversSameAreaAs(Path expectedPath, {required Rect areaToCompare, int sa
 // late Future<ByteData> someFont;
 // late WidgetTester tester;
 
-/// Asserts that a [Finder], [Future<ui.Image>], or [ui.Image] matches the
+/// Asserts that a [FinderBase], [Future<ui.Image>], or [ui.Image] matches the
 /// golden image file identified by [key], with an optional [version] number.
 ///
-/// For the case of a [Finder], the [Finder] must match exactly one widget and
+/// For the case of a [FinderBase], the [FinderBase] must match exactly one widget and
```

Comment:
Looks like it's a Finder:

https://github.com/flutter/flutter/blob/1fcd888704eadca5cd4b63b6dda50003d0913759/packages/flutter_test/lib/src/_matchers_io.dart#L93